evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -217,22 +232,81 @@ def predict(self, X, y=None):
             ValueError: If X was passed to `fit` but not passed in `predict`.
         """
         X, y = self._manage_woodwork(X, y)
-        fh_ = self._set_forecast(X)
-        X = X.ww.select(exclude=["Datetime"])
-        X.ww.set_types(
-            {
-                col: "Double"
-                for col in X.ww.select(["Boolean"], return_schema=True).columns
-            },
-        )
+        X, fh_ = self._manage_types_and_forecast(X=X)
+
         if not X.empty and self.use_covariates:
-            y_pred = self._component_obj.predict(fh=fh_, X=X)
+            y_pred_intervals = self._component_obj.predict_interval(
```

Comment:
@ParthivNaresh why do we want to use `predict_interval` here instead of `predict`?

Reply:
It's related to this Note [here](https://alteryx.atlassian.net/wiki/spaces/PS/pages/1531871378/Calculating+Prediction+Intervals+by+Estimator+in+Time+Series#Note). For `ARIMA` the predictions happen to be the mean of the prediction intervals, so instead of recalculating the prediction intervals for the most common interval of 95% confidence, I wanted to calculate it first and then save them, and return the predictions as their mean in order to save time later

Reply:
this sounds good to me! where are we exactly storing the results for 95 and reusing it though?